corehq/apps/locations/models.py

```diff
@@ -549,6 +550,43 @@ def _remove_user(self):
             user.active = False
             user.save()
 
+    def deprecate_by(self, other_location, operation, archive=True):
+        """
+        add metadata on locations
```

Comment:
I see `metadata` already exists on the location model; what's it currently used for? What are the tradeoffs of using metadata versus using custom location data fields?

Reply:
Those two are the same things
https://github.com/dimagi/commcare-hq/blob/d8d1e202a70f165e51243ccd097e205d233e6e50/corehq/apps/locations/forms.py#L432

and checked on [raw doc](https://india.commcarehq.org/hq/admin/raw_doc/?id=3b52cf7b2e924db9b992766c8c0c22aa&db_name=) for [this](https://india.commcarehq.org/a/icds-cas/settings/locations/3b52cf7b2e924db9b992766c8c0c22aa/) location

Reply:
nice, thank you for explaining